docs-content/general/6_product-features/6_metrics/2_metrics-tutorials/2_web-vitals-page-speed.md

```diff
@@ -0,0 +1,62 @@
+---
+toc: Web Vitals & Page Speed
+title: Creating Web Vitals & Page Speed Metrics
+slug: web-vitals-page-speed
+---
+
+<EmbeddedVideo 
+  src="https://www.youtube.com/embed/GMMdBR_61qw"
+  title="Metrics Tutorial: Measuring Page Speed & Web Vitals"
+  allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+/>
+
+## Overview
+
+This tutorial guides you through creating a graph to measure and analyze page speed over time. By following these steps, you'll be able to track your application's performance and gain insights into web vitals and other important metrics. Read more [here](../../../../getting-started/3_client-sdk/7_replay-configuration/opentelemetry.md) if you're interested in how Highlight's OpenTelemetry instrumentation captures this data, or [here](../../../../getting-started/2_frontend-backend-mapping.md) to learn more about how this is can relate to your backend instrumentation.
```

Comment:
Idea: Wonder if we can replace the "here"s with more explicit links? e.g.
```
Read more about [how Highlight's OpenTelemetry instrumentation captures this data](../../../../getting-started/3_client-sdk/7_replay-configuration/opentelemetry.md) and [how this is can relate to your backend instrumentation](../../../../getting-started/2_frontend-backend-mapping.md).
```
Or something of the sort.